src/Layer/LabelLayer.js

```diff
@@ -240,7 +242,7 @@ class LabelLayer extends GeometryLayer {
         // Converting the extent now is faster for further operation
         extent.as(data.crs, _extent);
         coord.crs = data.crs;
-        const globals = {
+        context.globals = {
```

Comment:
same here, could be done outside of this method

Reply:
becasue of zoom.extent it can't really be moved.